modules/Microsoft.Network/azureFirewalls/deploy.bicep

```diff
@@ -177,6 +178,33 @@ var diagnosticsMetrics = [for metric in diagnosticMetricsToEnable: {
   }
 }]
 
+var firewallProperties = azureSkuName == 'AZFW_VNet' ? {
```

Comment:
Why this variable? I'd suggest to keep it close to the resource

Reply:
Other than that I'd good with merging the PR. Thanks @fblix & @JPEasier

Reply:
good point. I guess it was a leftover from testing. removed it.